models/istio_validation.go

```diff
@@ -314,10 +318,12 @@ func (iv IstioValidations) MergeReferences(validations IstioValidations) IstioVa
 	return iv
 }
 
-func (iv IstioValidations) SummarizeValidation() IstioValidationSummary {
+func (iv IstioValidations) SummarizeValidation(ns string) IstioValidationSummary {
 	ivs := IstioValidationSummary{}
```

Comment:
maybe here there should be an explicit initialization? (see my comment on frontend PR)

Reply:
Default are 0 in this case. Then, the empty object looks like this:
```json
{
"errors": 0,
"objectCount": 0,
"warnings": 0
}
```
Does it make sense not to add any initial data?

Reply:
yep, that's fine ; tbh I'm sometimes puzzled with go handling of zero-value objects and i wasn't sure if that would come as 0 or undefined on serialization. It's fine to keep as you did.